test/integration/buffer_accounting_integration_test.cc

```diff
@@ -47,34 +47,6 @@ std::string protocolTestParamsAndBoolToString(
                                                : "without_per_stream_buffer_accounting");
 }
 
-void runOnWorkerThreadsAndWaitforCompletion(Server::Instance& server, std::function<void()> func) {
```

Comment:
this block was simply dead? 

Can you write a separate PR to remove it? Might as well not co-mingle this with your PR.